### ISSUE TITLE
After logging in : "ERROR: Please login before running this command"

### ISSUE THREAD

**Issue by tggagne** [ForceCLI/force#605]
> Droz2:Downloads tgagne$ f**orce login -i=test**
> Logged in as 'tgagne@hidden.gagne' (API v40.0)
> Your profile does not have Modify All Data enabled. Functionallity will be limited.
> Droz2:Downloads tgagne$ **force apiversion**
> ERROR: Please login before running this command.
> Droz2:Downloads tgagne$ **force active**
>  \-  - ns:

My user DOES have modify-all permission  This is with the latest version I just downloaded from both github and https://force-cli.herokuapp.com/

My sandbox has Financial Service Cloud installed--if it makes a difference.

**Comment by cwarden** [comment]
Thanks for the report, @tggagne.  The error suggests that the `describeMetadata` call performed after authentication is failing.
If you're able to help troubleshoot, here are builds that will dump the result of this call to stderr: https://gist.github.com/cwarden/ac9fa5cab7ae89dcb0b4edb35d890cb1

**Comment by tggagne** [comment]
The output was perfect.  I found the error.

When I downloaded it Friday and `chmod`ed it, I had to run it as root for mac to let me run it.  As a result (or maybe from a prior execution, ~/.force/current/account was owned by root instead of my user.

When I looked at the output from the version you sent I noticed an error about being unable to update the file above.  I chowned it back to the user and all is well.

Thank you!
